pyproject.toml

```diff
@@ -318,6 +319,27 @@ lint.ignore = [
     "PLE1205",  # Too many arguments for `logging` format string
     "RUF012",  # Mutable class attributes should be annotated with `typing.ClassVar`
     "RUF013",  # PEP 484 prohibits implicit `Optional`
+
+    # pydocstyle
+    # TODO: some of these will be enabled in their own patches
+    "D100",  # Missing docstring in public module
+    "D101",  # Missing docstring in public class
+    "D102",  # Missing docstring in public method
+    "D103",  # Missing docstring in public function
+    "D104",  # Missing docstring in public package
+    "D105",  # Missing docstring in magic method
+    "D106",  # Missing docstring in public nested class
+    "D107",  # Missing docstring in __init__
```

Comment:
Not sure which ones will remain ignored, but if for example the 101-107 are to be ignored, you could do "D10" to cover all of them.

Reply:
Hopefully, none of these would remain disabled :) We might convert them to in-file waibers, we might be missing too many docstrings that a patch fixing all of them would be huge and hard to review, ergo a file-by-file might be the best approach. I plan to experiment with these once the dust of this PR settles.

Reply:
Yes, I like the step-by-step approach. Listing all checks which are disabled for now makes sense.

Reply:
Oh, yes, I did not group them on purpose, so we could see their descriptions, and then enable them by dropping them from this list one by one.

Reply:
Understood. Like I wrote, the grouping would only be suitable for those that are to remain ignored